src/api/gtfsService.ts

```diff
@@ -118,9 +126,33 @@ export async function getGtfsStopHitTimesAsync(stop: BusStop, timestamp: Moment)
     .slice(0, MAX_HITS_COUNT)
 
   const rideIds = closestInTimeRides.map((ride) => ride.id).join(JOIN_SEPARATOR)
-  const stopHits = await GTFS_API.gtfsRideStopsListGet({
-    gtfsRideIds: rideIds,
-    gtfsStopIds: stop.stopId.toString(),
-  })
-  return stopHits.sort((hit1, hit2) => +hit1.arrivalTime! - +hit2.arrivalTime!)
+
+  const maxEndTime = Math.max(...rides.map((ride) => Number(ride.endTime)))
+  const minStartTime = Math.min(...rides.map((ride) => Number(ride.startTime)))
```

Comment:
just an idea - sometimes the actual ride happen before / after this range. maybe we should consider something like -
```suggestion
  const timeMargin = 1000 * 60 * 60 * 1.5 // 1.5 hours
  const maxEndTime = Math.max(...rides.map((ride) => Number(ride.endTime))) + timeMargin
  const minStartTime = Math.min(...rides.map((ride) => Number(ride.startTime))) + timeMargin
```

Reply:
Thanks for the quick response. I fixed the linter issues, added the correct type and added error & empty data error "handling" (can be improved) and submitted before seeing your replies. Would you like me to incorporate your suggestion and create another commit?

Reply:
It's up to you, it was just a suggestion.
Thanks for fixing the lint issues!